vignettes/markdown.Rmd

```diff
@@ -2,7 +2,14 @@
 title: "Write R Documentation in Markdown"
 author: "Gábor Csárdi"
 date: "`r Sys.Date()`"
-output: rmarkdown::html_vignette
+output:
+  html_document:
+    fig_caption: false
+    toc: true
+    toc_float:
+      collapsed: false
+      smooth_scroll: false
+    toc_depth: 3
```

Comment:
What's the benefit of `html_document` over `html_vignette`?

Reply:
Got it.

Reply:
I took the simpler approach and updated the Rpubs preview: http://rpubs.com/jennybc/432771